modules/external-services/rds.tf

```diff
@@ -0,0 +1,65 @@
+locals {
+  postgres_port = "5432"
+}
+
+data "aws_vpc" "selected" {
+  id = "${var.vpc_id}"
+}
+
+data "aws_subnet_ids" "rds" {
+  vpc_id = "${var.vpc_id}"
+  tags   = "${var.rds_subnet_tags}"
+}
+
+data "aws_subnet" "selected" {
+  count = "${length(data.aws_subnet_ids.rds.ids)}"
+  id    = "${data.aws_subnet_ids.rds.ids[count.index]}"
+}
+
+resource "aws_security_group" "db_access" {
+  description = "allow instances to talk to each other, and have unfettered egress"
+  vpc_id      = "${var.vpc_id}"
+
+  ingress {
+    protocol  = "tcp"
+    from_port = "${local.postgres_port}"
+    to_port   = "${local.postgres_port}"
+
+    cidr_blocks = ["${data.aws_subnet.selected.*.cidr_block}"]
+  }
+}
+
+resource "random_string" "database_password" {
+  length  = 40
+  special = false
+}
+
+resource "aws_db_subnet_group" "tfe" {
+  name       = "${var.prefix}tfe"
```

Comment:
as mentioned on the water cooler call today, ran into a collision with the already having a tfe group but hadn't set a prefix. (so mostly just leaving this as a reminder :D)